### PR TITLE
Update ruby, node and postgresql

### PR DESCRIPTION
**Pull request by simaishi**
The first PR to get master/ivanchuk up-to-date... Updated ruby, node and postgresql packages and a few minor cleanups. 

Trying to make just enough changes to get monolithic (which is built on top of podified) up and running first. So podified will still not work...

### CODE REVIEW

File: images/miq-app/Dockerfile [hunk 1]
```diff
@@ -53,8 +49,7 @@ RUN yum -y install centos-release-scl-rh \
                    openscap-scanner        \
                    patch                   \
                    psmisc                  \
-                   rh-postgresql95-postgresql-devel  \
-                   rh-postgresql95-postgresql-libs \
+                   postgresql-devel        \
```
Comment (carbonin) [inline images/miq-app/Dockerfile]: Is this versioned? Where are we getting this from?

Comment (simaishi) [reply images/miq-app/Dockerfile]: Our copr repo: https://copr.fedorainfracloud.org/coprs/manageiq/ManageIQ-Master/packages/

Comment (carbonin) [reply images/miq-app/Dockerfile]: Right, just found that ... do we want to rebuild it at some point? We're a few versions behind.

Oh, also we probably need to create an Ivanchuk repo, but this is fine we can merge this.

Comment (simaishi) [reply images/miq-app/Dockerfile]: Ivanchuk repo is already created. Didn't know we're already behind on version!!!  Will take a look.

Comment (carbonin) [reply images/miq-app/Dockerfile]: Yeah, think we're at 10.6 and 10.9 was released in June.

**Comment by simaishi** [comment]
Ivanchuk backport details:

```
$ git log -1
commit 96b196732e91d0a44b18b9021c723820e5d959f6
Author: Nick Carboni <ncarboni@redhat.com>
Date:   Fri Aug 2 15:47:03 2019 -0400

    Merge pull request #326 from simaishi/master_ivanchuk_updates
    
    Update ruby, node and postgresql
    
    (cherry picked from commit 4041a80af69ea473826d396e5f8635294a4858b3)
```

**Pull request merged**